docs/VnVPlan/VnVPlan.tex

```diff
@@ -486,10 +486,42 @@ \section*{Appendix --- Reflection}
   Examples of possible knowledge and skills include dynamic testing knowledge,
   static testing knowledge, specific tool usage etc.  You should look to
   identify at least one item for each team member.
+
+
+  \begin{itemize}
+    \item UI/UX usability validation tools such  as \textit{UserTesting}, \textit{Lookback.io}. to better evaluate our product is user-friendly in a couple of perspectives: effective, learnable, and user-friendly.
+    \item Dynamic Testing Tools such as \textit{Behave}, which is a tool that allows users to write the test cases in human languages to test for python-system framework. 
+    \item AI Model Validation Frameworks such as \textit{Snitch AI} which can help our trained morel enhance quality and troubleshoot quickly.
```

Comment:
I think for AI model in our project we probably would just use `scikit-learn` , but we could also try this framework :P

Reply:
Sure!!